internal/connectors/engine/workflow/terminate_schedules.go

```diff
@@ -34,15 +32,15 @@ func (w Workflow) runTerminateSchedules(
 			wg.Add(1)
 			workflow.Go(ctx, func(ctx workflow.Context) {
 				defer wg.Done()
-				// TODO(polo): context.Background() ?
-				scheduleHandler := w.temporalClient.ScheduleClient().GetHandle(context.Background(), s.ID)
-				if err := scheduleHandler.Delete(context.Background()); err != nil {
+
+				if err := activities.TemporalScheduleDelete(
+					infiniteRetryContext(ctx),
+					s.ID,
+				); err != nil {
```

Comment:
You might as well log the error in this PR instead of leaving a TODO

Reply:
Done thanks